PowerEditor/src/ScintillaComponent/Buffer.cpp

```diff
@@ -1347,6 +1357,31 @@ BufferID FileManager::newEmptyDocument()
 	return id;
 }
 
+BufferID FileManager::newPlaceholderDocument(const TCHAR* missingFilename, int whichOne)
+{
+	NppParameters& nppParamInst = NppParameters::getInstance();
+
+	static bool theWarningHasBeenGiven = false;
+	if (!theWarningHasBeenGiven)
+	{
+		int res = (nppParamInst.getNativeLangSpeaker())->messageBox(

```

Comment:
_I haven't actually tested it._
If res == IDNO - is there a reason to continue here and continue calling calling this function in `loadSession()`?

Reply:
Well caught!
Fix it: https://github.com/notepad-plus-plus/notepad-plus-plus/pull/14202/commits/ccf6f75c63e002730f6a76aefc85cdd0daa4d198